model/vsc_parser.py

```diff
@@ -17,7 +17,7 @@
 #
 # 2. Go through the tree and create a linked tree of Nodes (instances of
 #    typed classes).  Each node gets a more specific type than
-#    dict/list/string which can be used to evalute the tree contents later.
+#    dict/list/string which can be used to evaluate the tree contents later.
```

Comment:
@gunnarx - Should we add some copyright statements to the VSC py-files where we list contributors, similar to how it is handled in VSS files? Maybe we need to check most VSC-related files as I believe it is quite inconsistent at the moment.